suitcase/mongo_normalized/__init__.py

```diff
@@ -90,6 +92,23 @@ def __call__(self, name, doc):
         sanitized_doc = event_model.sanitize_doc(doc)
         return super().__call__(name, sanitized_doc)
 
+    def update(self, name, doc):
+        if name in ['run_start', 'run_stop']:
+            current_col = getattr(self, '_'+name+'_collection')
+            revisions_col = getattr(self, '_'+name+'_collection_revisions')
+            old = current_col.find_one({}, {'uid': doc['uid']})
+            old.pop('_id')
+            if revisions_col.count() == 0:
+                old['revision'] = 0
+            else:
+                cur = revisions_col.find().sort([('revision', -1)]).limit(1)
```

Comment:
This will find the revisions of every document, not just the one we are updating.